src/app/qgisapp.cpp

```diff
@@ -586,12 +587,11 @@ static void setTitleBarText_( QWidget &qgisApp )
     caption += QStringLiteral( " %1" ).arg( Qgis::devVersion() );
   }
 
+  // Add current profile (if it's not the only one)
   if ( QgisApp::instance()->userProfileManager()->allProfiles().count() > 1 )
   {
-    // add current profile (if it's not the default one)
```

Comment:
this was all intentional -- why the removal?

Reply:
Because with this PR, default profile does not mean quite the same thing, since the default profile can be choosen among the available profiles. So should we hide the profile name if it is "default" or rather if it is the current default one? 

But to be honest, I just did not get the point of hiding the profile name (unless it is the only one). I'd happily add it back if you think it is best ("default" or defaultProfileName though?)

Reply:
>  So should we hide the profile name if it is "default" or rather if it is the current default one?

I'd go with removal if it's the current default one. It just cleans up the application name shown in the taskbar with unnecessary extra information.